tests/difference/core/quint_model/ccv.qnt

```diff
@@ -0,0 +1,619 @@
+-*- mode: Bluespec; -*-
+module ccv {
+    import extraSpells.* from "./extraSpells"
+
+    // Things that are not modelled:
+    // * Reward distribution
+    // * Starting/Stopping chains during execution
+    // * Slashes
+
+    // Things that explicitly are modelled:
+    // * Validator set changes are propagated from provider to consumers
+    // * VSC packets mature
+    // * Consumers can be forcefully dropped due to timeouts, or stop on their own volition
+    
+    // TYPE DEFINITIONS
+    type Node = str
+    type Chain = str
+    type ValidatorSet = Node -> int
+    type Height = int
+    type Timestamp = int
+
+    // --PACKETS
+    type VSCPacket =
+        {   
+            // the identifier for this packet
+            id: int,
+            // the new validator set. in the implementation, this would be a list of validator updates
+            validatorSet: ValidatorSet,
+            // the time, that when passed on the receiver chain, will mean the packet is considered timed out
+            timeout: Timestamp
+        }
+
+    type VSCMaturedPacket = 
+        {
+            // the id of the VSCPacket that matured
+            id: int,
+            // the time, that when passed on the receiver chain, will mean the packet is considered timed out
+            timeout: Timestamp
+        }
+    
+    // MODEL PARAMETERS
+        
+
+    // set of identifiers of potential nodes
+    pure val Nodes: Set[Node] = 
+        Set("A", "B", "C", "D", "E", "F", "G", "H", "I", "J")
+
+    // the set of consumer chains
+    pure val ConsumerChains: Set[Chain] = 
+        Set("chain1", "chain2", "chain3")
+
+    // The singular provider chain.
+    pure val ProviderChain: Chain = 
+        "provider"
+        
+    pure val chains = ConsumerChains.union(Set(ProviderChain))
+
+    // length of the unbonding period on each chain
+    pure val UnbondingPeriod: Chain -> int = chains.mapBy(
```

Comment:
This is already a map of Chains to unbonding periods, i.e. it can be different per chain. Putting this into the state machine only makes sense if we want to test changing the unbonding period during runtime. Do you think that is a concern?